src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java

```diff
@@ -137,6 +147,7 @@ private static native long toDecimal(long nativeColumnView, boolean ansi_enabled
       int precision, int scale);
   private static native long toFloat(long nativeColumnView, boolean ansi_enabled, int dtype);
   private static native long fromDecimal(long nativeColumnView);
+  private static native long formatFloat(long nativeColumnView, int d);
```

Comment:
Other format function such as `fromDecimal` has a `from*` prefix so please also call the new function `fromFloat()`.